pkg/virt-controller/services/renderresources_test.go

```diff
@@ -431,6 +434,301 @@ var _ = Describe("Resource pod spec renderer", func() {
 	)
 })
 
+var _ = Describe("GetMemoryOverhead calculation", func() {
+	var (
+		vmi                     *v1.VirtualMachineInstance
+		staticOverhead          *resource.Quantity
+		baseOverhead            *resource.Quantity
+		coresOvehead            *resource.Quantity
+		videoRAMOverhead        *resource.Quantity
+		cpuArchOverhead         *resource.Quantity
+		vfioOverhead            *resource.Quantity
+		downwardmetricsOverhead *resource.Quantity
+		sevOverhead             *resource.Quantity
+		tpmOverhead             *resource.Quantity
+		passtOverhead           *resource.Quantity
+	)
+
+	BeforeEach(func() {
+		vmiMemoryReq := resource.MustParse("1Gi")
+		vmi = &v1.VirtualMachineInstance{
+			ObjectMeta: metav1.ObjectMeta{Name: "test-vmi"},
+			Spec: v1.VirtualMachineInstanceSpec{
+				Domain: v1.DomainSpec{
+					Resources: v1.ResourceRequirements{
+						Requests: kubev1.ResourceList{
+							kubev1.ResourceMemory: vmiMemoryReq,
+						},
+						Limits: kubev1.ResourceList{},
+					},
+				},
+			},
+		}
+		staticOverhead = resource.NewScaledQuantity(0, resource.Kilo)
+		staticOverhead.Add(resource.MustParse(VirtLauncherMonitorOverhead))
+		staticOverhead.Add(resource.MustParse(VirtLauncherOverhead))
+		staticOverhead.Add(resource.MustParse(VirtlogdOverhead))
+		staticOverhead.Add(resource.MustParse(VirtqemudOverhead))
+		staticOverhead.Add(resource.MustParse(QemuOverhead))
+		// static overhead for IOThread
+		staticOverhead.Add(resource.MustParse("8Mi"))
+
```

Comment:
I think you can simplify and just use constant value. This feels like duplicating the production logic